assets/sass/components/_header.scss

```diff
@@ -0,0 +1,77 @@
+.site-header {
+  padding-top: 60px;
+}
+.site-title {
```

Comment:
@nucro we should change the markup in the header so that we can follow the BEM guidelines more closely with our SCSS.  For example, the header has a class of "site-header", then there is a div in the header with the class "site-branding", a wrapper around the <h1> with a class of "site-title", and then a div with a class named "site-description"  The "site-title" div around the <h1> should be removed, and the format of the class names should be changed to this:

<header class="header">
 <div class="header__branding">
  <h1 class="header__branding__title">
  <div class="header__branding__description">

Then you can use BEM in your SCSS and the code would be like so:

.site-header{
  padding-top: 60px;
  &__branding{
  /* styles here */
    &__title{
      /* h1 styles here */
    }
    &__description{
      /* .site-description styles here */
    }
  }
}

please see the header.php file here https://github.com/moxie-leean/Leean/blob/master/header.php

Let me know if you have any questions

cc/ @mitogh